base/cvd/cuttlefish/host/commands/assemble_cvd/bootconfig_args.cpp

```diff
@@ -210,6 +210,21 @@ Result<std::unordered_map<std::string, std::string>> BootconfigArgsFromConfig(
           ? "com.android.hardware.gatekeeper.nonsecure"
           : "com.android.hardware.gatekeeper.cf_remote";
 
+  // jcardsim
```

Comment:
nit: let's use full name: `jcardsimulator` rather than `jcardsim` here in other places  in the PR to avoid confusion with telephony sim card.

Reply:
Changed the name to jcardsimulator